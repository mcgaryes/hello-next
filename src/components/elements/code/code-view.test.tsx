import {render} from '@testing-library/react'
import CodeView from "@/elements/code/code-view";

test('Renders code-view unchanged', () => {
    const {container} = render(<CodeView value={{}}/>)
    expect(container).toMatchSnapshot()
})
