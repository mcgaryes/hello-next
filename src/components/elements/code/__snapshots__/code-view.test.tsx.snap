// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders code-view unchanged 1`] = `
<div>
  <div
    class="overflow-scroll rounded-2xl border-gray-200 border bg-gray-50 p-8 text-sm"
  >
    <pre>
      {}
    </pre>
  </div>
</div>
`;
